_data/sites.json

```
@@ -6581,6 +6581,16 @@
         ]
     },
 
+    {
+        "name": "Flat",
+        "url": "https://flat.io/settings/account/profile",
+        "difficulty": "easy",
+        "notes": "Log-in, go to the link and click the **Delete my account** button",
+        "domains": [
+            "flat.io"
```

There is another domain:

```suggestion
            "flat.io",
            "help.flat.io"
```